lib/matplotlib/streamplot.py

```diff
@@ -56,6 +57,10 @@ def streamplot(axes, x, y, u, v, density=1, linewidth=None, color=None,
     arrowstyle : str
         Arrow style specification.
         See `~matplotlib.patches.FancyArrowPatch`.
+    numarrows : int
+        Number of arrows per streamline.
+        The default is 1, which plots one arrow in the middle of the
+        streamline.
```

Comment:
```suggestion
    numarrows : int, default: 1
        Number of arrows per streamline. The arrows are
        distributed equidistantly on each line.
```
I think (did not follow all details of the code).